cove_bods/views.py

```diff
@@ -89,6 +89,12 @@ def explore_bods(request, pk):
         db_data.rendered = True
     db_data.save()
 
+    # We need to calculate some stats for showing in the view
+    total_interest_statements = 0
```

Comment:
I'm slightly confused about "interest statements". Aren't interests an array within ownership-or-control statements, rather than a statement themselves?

Reply:
Yeah, that could be a better variable name.

Reply:
Its also called that in the web UI.

Reply:
It is called that in the Web UI, but under a header that says "Ownership or Control Statements" so I thought that was ok. Is this a detail we can pick up in the redesign that the next sprint will be around?